debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDatabaseSchema.java

```diff
@@ -214,7 +214,7 @@ private List<SchemaChangeEvent> parseDdl(MySqlPartition partition, String ddlSta
         }
         catch (ParsingException | MultipleParsingExceptions e) {
             if (schemaHistory.skipUnparseableDdlStatements()) {
-                LOGGER.warn("Ignoring unparseable DDL statement '{}': {}", ddlStatements, e);
+                LOGGER.warn("Ignoring unparseable DDL statement '{}'", ddlStatements, e);
```

Comment:
@jpechane didn't this remove printing the exception? :thinking:

Reply:
> didn't this remove printing the exception?

@rk3rn3r No. If the last parameter is a `Throwable`, SLF4J (and Logback) will use it to print the exception cause/stack trace.

> The SLF4J API supports parametrization (of 1 or more objects) in the presence of an exception, assuming the exception is the last parameter.

https://www.slf4j.org/faq.html#paramException